mvr-cli/src/lib.rs

```diff
@@ -1226,3 +1250,45 @@ pub fn sui_config_dir() -> Result<PathBuf, anyhow::Error> {
         Ok(dir)
     })
 }
+
+/// Migrates the lockfile to version 3 from version 2, if necessary.
+fn migrate_to_version_three(packages: &mut ArrayOfTables) {
+    for package in packages.iter_mut() {
+        // if ID is missing from the lockfile, add it.
+        if !package.contains_key(LOCK_PACKAGE_ID_KEY) {
+            package.insert(
+                LOCK_PACKAGE_ID_KEY,
+                package.get(LOCK_PACKAGE_NAME_KEY).unwrap().clone(),
+            );
+        }
+
+        if !package.contains_key(DEPENDENCIES_KEY) {
+            continue;
+        }
+
+        // SAFETY: We've already checked that `dependencies` is part of the package table.
+        let dependencies = package.get_mut(DEPENDENCIES_KEY).unwrap();
+
+        if !dependencies.is_array() {
+            continue;
+        }
+
+        // SAFETY: We've already checked that `dependencies` is an array.
+        for dep in dependencies.as_array_mut().unwrap().iter_mut() {
+            let val = dep.as_inline_table_mut().unwrap();
+            if !val.contains_key(LOCK_PACKAGE_ID_KEY) {
+                val.insert(
+                    LOCK_PACKAGE_ID_KEY,
+                    val.get(LOCK_PACKAGE_NAME_KEY).unwrap().clone(),
+                );
+            }
+
+            if !val.contains_key(LOCK_PACKAGE_NAME_KEY) {
+                val.insert(
+                    LOCK_PACKAGE_NAME_KEY,
+                    val.get(LOCK_PACKAGE_ID_KEY).unwrap().clone(),
+                );
+            }
+        }
+    }
```

Comment:
Can we rework the unwraps? I know they're annoying but let's try to keep the code as clean as we can and minimize risks for panics.